### PR TITLE
subversion: bump to the latest stable bugfix release

### PR DESCRIPTION
**Pull request by val-kulkov**
Maintainer: me
Compile tested: arm_cortex-a9, Buffalo WXR-1900DHP, OpenWrt SNAPSHOT r8214-7bfe757
Run tested: arm_cortex-a9, Buffalo WXR-1900DHP, OpenWrt SNAPSHOT r7274-56f3aee

Description:
A straightforward update to v1.10.3.

Signed-off-by: Val Kulkov <val.kulkov@gmail.com>

### CODE REVIEW

**Comment by hnyman** [comment]
What does the global travis change do in subversion PR?
@champtar or @lynxis might have views on that

**Comment by yousong** [comment]
I downloaded the complete log and maybe the following lines tell the direct cause.

```
https://api.travis-ci.org/v3/job/440346983/log.txt

12030 checking for library containing shmget... none required^M
12031 checking for inflate in -lz... no^M
12032 configure: error: zlib library not found^M
12033 If you have zlib already installed, see config.log for details on the^M
12034 failure.  It is possible the compiler isn't looking in the proper directory.^M
12035 Use --without-zlib to disable zlib support.^M
12036 make[2]: *** [/tmp/tmp.SvOP0RoGEX/build_dir/hostpkg/postgresql-9.6.10/.configured] Error 1^M
12037 make[2]: Leaving directory `/home/travis/build/openwrt/packages/libs/postgresql'^M
12038 time: package/feeds/packages/postgresql/host-compile#11.63#2.12#16.06^M
12039 make[1]: *** [package/feeds/packages/postgresql/host/compile] Error 2^M
12040 make[1]: Leaving directory `/tmp/tmp.SvOP0RoGEX'^M
12041 make: *** [package/subversion/compile] Error 2^M
12042 ^[[1;31m=> 'make package/subversion/compile -j1 V=s' failed (return code 2)^[[m^M
12043 ^[[1;34m=== subversion: compile test done^[[m^M
```

**Comment by yousong** [comment]
I guess the conditional dependency on unixodbc triggers host build of postgresql (unixodbc selects libpq).  @val-kulkov you may consider adding a test commit removing ` +PACKAGE_unixodbc:unixodbc` from subversion

the zlib prereq check was moved from include/prereq-build.mk to tools/zlib/ .  Probably this will also need to be resolved.

**Comment by val-kulkov** [comment]
@yousong : yes, thank you, I am looking into the zlib dependency and I am going to check if the dependency on unixodbc can be somehow avoided. The fact that subversion compiles successfully on my host/target combination with my custom .config file certainly does not help me. At this point, I think I should create a clean build system with the default .config locally and attempt to compile subversion using the default .config.

**Comment by yousong** [comment]
@val-kulkov , note that in this case the availability of zlib devel package is the key.  OpenWrt build system will have zlib build as a prereq while it looks like the sdk has no such facility in place.

**Comment by val-kulkov** [comment]
@yousong : hmm, I'll see if PKG_BUILD_DEPENDS on zlib solves the problem. Thank you!

File: net/subversion/Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ PKG_MAINTAINER:=Val Kulkov <val.kulkov@gmail.com>
 
 PKG_FIXUP:=autoreconf
 PKG_MACRO_PATHS:=build/ac-macros
-PKG_BUILD_DEPENDS:=apr-util
+PKG_BUILD_DEPENDS:=apr-util zlib
```
Comment (yousong) [inline net/subversion/Makefile]: I am afraid it's host zlib library that postgresql host build is looking for ;(

**Comment by val-kulkov** [comment]
Apologies for the delay to everyone who commented here. I have not yet had a chance to research what's going on here with the dependencies. I may not have enough spare time to solve this issue in the next week or two. Please bear with me and do not consider this PR abandoned.

**Comment by hnyman** [comment]
Any progress here?

(And please remove the travis changes from the PR)

**Comment by val-kulkov** [comment]
@hnyman : since v1.11.0 is already out, I am going to close this PR and create a new one for v.1.11.0; but I want to find out first what is going wrong here.

**Comment by val-kulkov** [comment]
This PR is not relevant anymore. Subversion v.1.10.3 has been superseded by v.1.11.0.

**Pull request closed without merging**